modules/service-bus/namespace/main.bicep

```diff
@@ -170,16 +207,22 @@ resource cMKKeyVaultKey 'Microsoft.KeyVault/vaults/keys@2021-10-01' existing = i
   scope: resourceGroup(split(cMKKeyVaultResourceId, '/')[2], split(cMKKeyVaultResourceId, '/')[4])
 }
 
-resource serviceBusNamespace 'Microsoft.ServiceBus/namespaces@2021-11-01' = {
+resource serviceBusNamespace 'Microsoft.ServiceBus/namespaces@2022-10-01-preview' = {
   name: name
   location: location
   tags: empty(tags) ? null : tags
   sku: {
     name: skuName
+    capacity: skuName == 'Premium' ? skuCapacity : null
   }
   identity: identity
   properties: {
+    publicNetworkAccess: !empty(publicNetworkAccess) ? publicNetworkAccess : (!empty(privateEndpoints) && empty(networkRuleSets) ? 'Disabled' : 'Enabled')
```

Comment:
I think this would not be WAF aligned and the default case would need to be 'Disabled'. It's also something we can address in AVM, but I just had the case that PSRule slammed that in my face for CognitiveServices.

Reply:
@AlexanderSehr at the moment its a mix bag in CARML modules of the default state. Some are not set to `disabled`, some are left `''`, some are `Enabled`. I believe we can address this in AVM as modules need to comply with WAF and have waf specific use cases.

Reply:
Fair point. Just wanted to bring it up as I just had the case. Will resolve the issue